connectors/spark/src/main/scala/io/unitycatalog/spark/UCHadoopConf.java

```diff
@@ -20,6 +20,11 @@ private UCHadoopConf() {
   public static final String AZURE_INIT_SAS_TOKEN_EXPIRED_TIME =
       "fs.azure.init.sas.token.expired.time";
 
+  // Keys for the initialized Google Cloud Storage OAuth token.
+  public static final String GCS_INIT_OAUTH_TOKEN = "fs.gs.init.oauth.token";
+  public static final String GCS_INIT_OAUTH_TOKEN_EXPIRED_TIME =
+      "fs.gs.init.oauth.token.expired.time";
```

Comment:
Nit: Maybe we need to correct the name, the correct key should be `fs.gs.init.oauth.token.expiration.time`, this also applies to `fs.s3a.init.credential.expiration.time`.